tests/test_indexer.py

```diff
@@ -327,6 +347,65 @@ def get_search_results(cls, query, expected_hit_count):
                 time.sleep(0.5)
 
 
+class BundleBuilder:
```

Comment:
Added a class docstring. It is brief, given this is a test class used within a single test file.